### DISCUSSION TITLE
Does this repo will support .net 6.0 (MAUI)

### DISCUSSION THREAD

**Discussion by dingxiaolong** [PrismLibrary/Prism#2695] [.NET MAUI] [answered]
Dear :
As Microsoft not support Xamarin.form next year  .
So do this repo will support the new .net6.0 or MAUI in the future ?

Hoping your response 
Thanks . Br.

**Comment by dansiegel** [comment, marked as answer]
Yes we are supporting .NET MAUI. Currently preview packages are available for Sponsors. You can follow the work at [dansiegel/Prism.Maui](/dansiegel/Prism.Maui). Once the work has been stabilized we will be moving the code into this repo

**Reply by dingxiaolong** [reply]
Thanks very much.
